content/getting-started/index.mdx

```diff
@@ -8,9 +8,11 @@ To use Chakra UI in your project, run the following command in your terminal:
 
 ```bash
 npm i @chakra-ui/react @emotion/react @emotion/styled framer-motion
+```
 
-# or
+or
 
+```
```

Comment:
I would also add that this second code block also needs the `bash` keyword so that this line for the yarn install is highlighted correctly.

` ```bash `

Reply:
Thanks for the suggestion. Implemented it!